CHANGELOG.md

```diff
@@ -34,6 +34,36 @@ New features:
   Now, these types' kind signatures are displayed above their declarations
   in their docs, similar to what one would see in the source code.
 
+  But that's not all!
+
+  The two below types do not have explicit kind signatures, but the compiler
+  infers what their kind signature is anyway. "Interesting" kind signatures
+  will be displayed but "uninteresting" kind signatures (even if they are
+  explicit) will be removed.
+
+  ```purescript
+                                                  {-
+  data Foo :: (Type -> Type) -> Type -> Type      -}
+  data Foo f a = Foo (f a)
+                                                  {-
+  data Tuple :: Type -> Type -> Type              -}
+  data Tuple a b = Tuple a b
+  ```
+
+  The docs will include the kind signature for `Foo`, but will
+  intentionally remove the kind signature for `Bar`, even if `Bar`'s
```

Comment:
Bar or Tuple?

Reply:
Ah! Ran out of time again. Thanks for pointing that out.